torch/distributed/nn/jit/instantiator.py

```diff
@@ -62,20 +62,6 @@ def get_arg_return_types_from_interface(module_interface):
     return args_str, arg_types_str, return_type_str
 
 
-@atexit.register
-def cleanup_generated_modules():
```

Comment:
where does this function go? Looks like test is still calling this?

Reply:
This is handled by destructing of `tempfile.TemporaryDirectory`. Just updated the tests.

Reply:
Discussed with @xush6528 offline. This is now handled by `TemporaryDirectory()`.

And it will live until the script exists (see https://stackoverflow.com/questions/17301091/python-lifetime-of-module-global-variables). This should be sufficient for now. If we want to delete it earlier, we can add some APIs for that later.